.gitignore

```diff
@@ -46,4 +46,4 @@ sweep.timestamp
 
 crates/turborepo-ffi/bindings.h
 crates/turborepo-ffi/ffi/proto/*
-cli/internal/ffi/libturborepo_ffi.a
```

Comment:
We should probably add this as ignored to `CODEOWNERS`

Reply:
Does `CODEOWNERS` care about gitignore'd files?

Reply:
I meant the `.gitignore` file itself, sorry 🙈